uelc/main/views.py

```diff
@@ -99,7 +100,17 @@ def perform_checks(self, request, path):
                                    extra_tags='rootUrlError')
                     request.path = '/'
                     return HttpResponseRedirect(request.path)
+
             return visit_root(self.section, self.no_root_fallback_url)
+
+        if self.section == self.module and pt == "group_user":
+            '''forward them to the home page of the part'''
+
+            ns_path = hierarchy.base_url + self.section.get_next().get_path()
```

Comment:
This can work but to avoid cases where there's missing or duplicate slashes, it might be better to concat this url like `urlparse.urljoin(hierarchy.bash_url, self.section.get_next().get_path())`
See: https://github.com/ccnmtl/dmt/blob/master/bdd_tests/environment.py#L19